recipe/meta.yaml

```diff
@@ -1,18 +1,18 @@
-{% set version = "3.6.5" %}
+{% set version = "3.7" %}
 
 package:
   name: nltk
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/n/nltk/nltk-{{ version }}.zip
-  sha256: 834d1a8e38496369390be699be9bca4f2a0f2175b50327272b2ec7a98ffda2a0
+  sha256: d6507d6460cec76d70afea4242a226a7542f85c669177b9c7f562b7cf1b05502
 
 build:
   number: 0
-  skip: True  # [py<36]
+  skip: True  # [py<37]
   noarch: python
```

Comment:
we probably should eliminate the `skip` section since we already have the python pinning restriction in the `run` section

https://github.com/AnacondaRecipes/nltk-feedstock/blob/pbs_3.7_20220210T110752/recipe/meta.yaml#L13

Reply:
I removed it